.travis.yml

```diff
@@ -44,6 +44,7 @@ script:
   - cd tests
   - python3 -m unittest test_randomizedsearch.py
   - python3 -m unittest test_gridsearch.py
+  - cd ~/ && git clone https://github.com/ray-project/ray && python3 ray/doc/source/tune/_tutorials/tune-sklearn.py
```

Comment:
I think this might mess with the next line where we run stuff in the examples folder because `../examples` will be the wrong path. We could just move this line down to after the examples stuff.